datumaro/plugins/shift_analyzer.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import cv2
+import numpy as np
+
+from datumaro.components.errors import MediaTypeError
+from datumaro.components.media import Image
+from datumaro.plugins.openvino_plugin.launcher import OpenvinoLauncher
+
+
+def img_center_crop(image, size):
+    width, height = image.shape[1], image.shape[0]
+    mid_w, mid_h = int(width / 2), int(height / 2)
+
+    crop_w = size if size < image.shape[1] else image.shape[1]
+    crop_h = size if size < image.shape[0] else image.shape[0]
+    mid_cw, mid_ch = int(crop_w / 2), int(crop_h / 2)
+
+    cropped_image = image[mid_h - mid_ch : mid_h + mid_ch, mid_w - mid_cw : mid_w + mid_cw]
+    return cropped_image
+
+
+def img_normalize(image):
+    mean = np.array([127.5] * 3)
+    std = np.array([127.5] * 3)
+
+    image = image.transpose(-1, 0, 1)
+    image = (image - mean[:, None, None]) / std[:, None, None]
+    return image
```

Comment:
I think those are input pre-processing and they have to reside in `InterpreterScript` because each model has its own settings. For example, `datumaro/plugins/openvino_plugin/samples/googlenet-v4-tf_interp.py` can have `preprocess()` function and we can move those line to that function.